phpunit/functional/ITILSolutionTest.php

```diff
@@ -634,4 +641,43 @@ public function testAddEmptyContent()
         ]);
         $this->assertGreaterThan(0, $solution_id);
     }
+
+    public function testSendSolutionWithMandatoryFields()
```

Comment:
Unless I missed something, you do not test a missing mandatory field throws an error

Reply:
When I test if it raises an error it works but the test fails because an exception is thrown, so I did the opposite (check that it doesn't raise an error).

![image](https://github.com/user-attachments/assets/58b3fadd-4706-4cd1-b64a-b899123e6c76)

Reply:
In that kind of case, you have to test both situations: mandatory field filled gives no error, and mandatory field missing give an error. Look at other tests files, there already have example of tests with exceptions or other kind of errors (some just log a wanring in the logs, etc).